guidelines.md

```diff
@@ -18,29 +38,42 @@ All plots **must**:
   See [recommendations](#recommendations) below for further advice on what this might mean in practice.
 - **Be labelled with the appropriate watermark** (typically one of the following:
   DUNE, DUNE Preliminary, DUNE Work In Progress, DUNE Simulation).
-- **Be saved in at least in one lossless format** (ideally vector-based---e.g., pdf or eps),  
-  and **at least one high-quality raster-based format** (e.g., .png).
+- **Be saved in at least in one vector-based format** (e.g., pdf or eps),  
+  and **at least one high-quality raster-based format** (e.g., png).
 
 ## Recommendations
+These items listed here are recommendations, but not strictly enforced. Plots should be internally consistent within documents, and other considerations or strong personal preferences can supersede the recommendations.
```

Comment:
I think this is redundant with the text added in lines 18-20 about what "requirements" vs "recommendations" mean.  I'm hesitant to include _more_ disclaimers here, because it will dilute the strength of our recommendations (we really do want people to use them in general!).  Maybe we should move the "internally consistent" part to a "Requirement" though?

Reply:
Very fair, I'd added this when we were both editing and didn't catch the duplication on a read through, we can remove. Agreed on making internal consistency a requirement.

Reply:
Done!